toponetx/classes/cell_complex.py

```diff
@@ -130,10 +130,10 @@ class CellComplex(Complex):
     >>> CX.is_regular
     """
 
-    def __init__(
-        self, cells=None, name: str = "", regular: bool = True, **kwargs
-    ) -> None:
-        super().__init__(name, **kwargs)
+    def __init__(self, cells=None, name: str = "", regular=True, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+        self.name = name
```

Comment:
```suggestion
        super().__init__(name, **kwargs)
```
(No need to set `self.name` if passing the name to the superclass)